cypress/fixtures/modules/display.json

```diff
@@ -0,0 +1,68 @@
+{
+    "COLORDEPTH": 10,
+    "COLORIMETRY": ["BT2020RGB","BT2020YCC","xvYCC601", "xvYCC709"],
+    "HDRPROFILES": ["dolbyVision","hdr10","hdr10plus","hlg"],
+    "MANUFACTURER": "Samsung Electric Company",
+    "PRODUCTNAME": "Q80A",
+    "REFRESHRATE": 60,
+    "RESOLUTION": [1920,1080],
+    "RESOLUTIONNAME": "uhd",
+    "SIZE": [157,91],
+    "SOURCEPHYSICALADDRESS":  "3.0.0.0",
+    "ONCOLORDEPTHCHANGED_EVENT":{
+        "method": "display.onColorDepthChanged",
+        "result": 10 
+    },
+    "ONCOLORIMETRYCHANGED_EVENT":{
+        "method": "display.onColorimetryChanged",
+        "result": [
+            "BT2020RGB",
+            "BT2020YCC",
+            "xvYCC601",
+            "xvYCC709"
+        ]
```

Comment:
these must be added in MFOS config module mac.json file. These would be specific to device

Reply:
Moved to mac.json file of MFOS config module